tests/_support/DatabaseTestCase.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Tests\Support;
+
+use CodeIgniter\Test\CIUnitTestCase;
+use CodeIgniter\Test\DatabaseTestTrait;
+
+/**
+ * @internal
+ */
+abstract class DatabaseTestCase extends CIUnitTestCase
```

Comment:
This should extend `Tests\Support\TestCase`.

Reply:
Done.